docs/source/languages.rst

```diff
@@ -3,30 +3,31 @@
 Supported languages
 ===================
 
-Snips NLU supports various languages, that are speficied in the dataset in the
-``"language"`` attribute. Here is the list of supported language along with
-their isocode:
+Snips NLU supports various languages. The language is specified in the
+:ref:`dataset <json_format>` in the ``"language"`` attribute.
 
-+-----------------+------------+
-| Language        | ISO code   |
-+=================+============+
-| German          | de         |
-+-----------------+------------+
-| English         | en         |
-+-----------------+------------+
-| Spanish         | es         |
-+-----------------+------------+
-| French          | fr         |
-+-----------------+------------+
-| Italian         | it         |
-+-----------------+------------+
-| Japanese        | ja         |
-+-----------------+------------+
-| Korean          | ko         |
-+-----------------+------------+
-| Portuguese (BR) | pt_br      |
-+-----------------+------------+
-| Portuguese (PT) | pt_pt      |
-+-----------------+------------+
+Here is the list of the supported languages along with their isocode:
+
++------------+
```

Comment:
It seems that we've lost the first column is it normal ?

Reply:
Yes, the language names are not exposed yet by the ontology ffi. I will add it once it's available, but I think we can move forward with this version for now.